balancer_switching_test.go

```diff
@@ -467,3 +467,52 @@ func TestSwitchBalancerGRPCLBServiceConfig(t *testing.T) {
 		t.Fatalf("after 5 second, cc.balancer is of type %v, not round_robin", cc.curBalancerName)
 	}
 }
+
+// Test that when switching to grpclb fails because grpclb is not registered,
+// the fallback balancer will only get backend addresses, not the grpclb server
+// address.
+//
+// The tests sends 3 server addresses (all backends) as resolved addresses, but
+// claim the first one is grpclb server. The all RPCs should all be send to the
+// other addresses, not the first one.
+func TestSwitchBalancerGRPCLBWithGRPCLBNotRegistered(t *testing.T) {
+	balancer.UnregisterForTesting("grpclb")
+	defer balancer.Register(&magicalLB{})
+
+	defer leakcheck.Check(t)
+	r, rcleanup := manual.GenerateAndRegisterManualResolver()
+	defer rcleanup()
+
+	numServers := 3
```

Comment:
`const`  (or just delete and use the 3 directly on the following line)

Reply:
Done.